### ISSUE TITLE
File::Find::fullname is always undef on Windows 10

### ISSUE THREAD

**Issue by jimav** [Perl/perl5#21143]
If File::Find is run with `follow => 1` on Windows, `$File::Find::fullname` is always undef.

As the docs say, _"follow ... is also a no-op on Win32"_, literally, i.e. $fullname is not implemented at all and so is always undef.

This seems unsafe because scripts written to handle symlinks **must** check File::Find::fullname for undef to avoid wrongly following broken links, and as a result such scripts think _every_ file is a broken link on Windows.

I'm wondering if it would be better to set `fullname` to `name` on Win32 (and also guarantee to stat the file so the "_" handle is valid) so scripts can more often work as-is without having to hack around with $^O in the `wanted` function.

**Comment by xenu** [comment]
`follow => 1` isn't actually a no-op on Windows since Perl 5.34, the docs were wrong (it's fixed in blead).

**Comment by jkeenan** [comment]
> `follow => 1` isn't actually a no-op on Windows since Perl 5.34, the docs were wrong (it's fixed in blead).

Yes, see ac083dce6a86105e47992498ab05965aae99496b, committed Aug 2 2022.  Closing this ticket.

**Comment by jimav** [comment]
@jkeenan - that commit only changes the pod.  Was something else also changed so that `$File::Find::fullname` isn't always undef on Windows?

**Comment by jkeenan** [comment]
> @jkeenan - that commit only changes the pod. Was something else also changed so that `$File::Find::fullname` isn't always undef on Windows?

@xenu, can you comment on the above?
